requirements.txt

```diff
@@ -10,13 +11,7 @@ bitsandbytes>=0.41.1
 evaluate>=0.4.0
 tokenizers>=0.13.3
 protobuf
-# Transformers library (v4.34.0) still has a bug for left padding, 
-# and significantly affect the inference and thus our evaluation performance (e.g., MMLU and TruthfulQA).
-# Follwing PR is a temporary fix for it but has not been merged yet.
-# See https://github.com/huggingface/transformers/pull/25284
-# But this PR is not compatible with the latest version of Transformers library (v4.34.0).
-# To incorporate it, we forked the Transformers library and made some changes to make it compatible with the latest version.
-git+https://github.com/yizhongw/transformers.git@left_padding
+transformers==4.40
```

Comment:
I pinned the transformer version to my own fork before, as there is a bug in the left padding that affects batch inference performance. See [here](https://github.com/allenai/open-instruct/blob/main/requirements.txt#L13-L19).

I am not sure whether this is still an issue. @hamishivi any idea here? It's better to test it using MMLU scripts with an eval_batch_size > 1.

Reply:
Let me test this!

Reply:
tested and this is no longer an issue.